### ISSUE TITLE
Create a guide/section detailing how to map IPFS hashes to decentralized name systems

### ISSUE THREAD

**Issue by realChainLife** [ipfs/ipfs-docs#411]
Now that go-ipfs supports the resolution of DNSLink for `.eth` names through the EthDNS can we add a guide for users explaining: 

- how EthDNS work with IPFS
- how to add a content hash to domain addresses. 

We can also add guides for `.crypto` and `.zil` addresses. Perhaps we can add it to the `/concepts/dnslinks` docs section.

**Comment by johnnymatthews** [comment]
I agree, this sounds like it would be a useful guide to have. It might also be useful to attach it to the [Websites on IPFS](https://docs.ipfs.io/how-to/websites-on-ipfs/single-page-website/) section.

Would you say ENSLink deserves it's own page, or can it be combined with the [DNSLink page](https://docs.ipfs.io/concepts/dnslink/)? I've not looked into ENSLink thoroughly yet, so I'm relying on your knowledge of both ENSLink and DNSLink here @realChainLife.

**Comment by realChainLife** [comment]
Hey @johnnymatthews yes, we can add some pointers on the Websites on IPFS: Set up a domain section. 

We could also have an EthDNS, EthLink specific page that's a deep dive on how EthDNS works with IPFS, but if it doesn't make the write-up too long we can add the giuideline to set contenthash to ENS to the Alternatives to IPNS section of the updated [IPNS concept doc](https://github.com/ipfs/ipfs-docs/issues/244).

**Comment by BlocksOnAChain** [comment]
Documentation for https://dnslink.io/ and https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#dns
